src/pks/transport/transport_ats_pk.cc

```diff
@@ -175,7 +173,8 @@ void Transport_ATS::Setup(const Teuchos::Ptr<State>& S)
   S->RequireField(solid_residue_mass_key_,  name_, subfield_names)
     ->SetMesh(mesh_)->SetGhosted(true)
     ->SetComponent("cell", AmanziMesh::CELL, ncomponents);
-
+  
```

Comment:
no need for extra whitespace please (though this kind of thing should get cleaned up soon in the state PR)

Reply:
@ecoon I though we decided to close state PR first and then push a cleaning patch with supporting Python scripts.

Reply:
yes, that's what I should have said -- it will be cleaned up soon after the state PR